lib/maplibre-compose/src/androidMain/kotlin/dev/sargunv/maplibrecompose/compose/AndroidMapView.kt

```diff
@@ -68,6 +69,7 @@ internal fun AndroidMapView(
               mapView = mapView,
               map = map,
               scaleBar = AndroidScaleBar(context, mapView, map),
+              mapSnapshotter = AndroidMapSnapshotter(context, layoutDir, density),
```

Comment:
we need to make sure layout dir and density gets updated when it changes (see below in the `update` callback)